packages/p2p/src/components/my-ads/my-ads.scss

```diff
@@ -199,6 +199,14 @@
         flex-direction: column;
         padding: 1rem 1.6rem 0;
         width: 100vw;
+
+        &--scroll {
+            overflow: auto;
+        }
```

Comment:
Removed `margin-bottom: 7.5rem` as it is a bug added in previous PR which will be fixed in card Task #64742